65black/StackOlite
# #160710315 Search For A Question

#### What does this PR do?
This PR consumes the `POST /api/v1/search` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Click on the search input in the header, type in your search term and click on the search icon.
 - A modal should pop up with the result of your search query.

#### Screenshots
![search](https://user-images.githubusercontent.com/36507972/46137503-ae6f6980-c241-11e8-8e8b-d4c847801c3c.JPG)

#### What are the relevant pivotal tracker stories?
#160710315
